### ISSUE TITLE
Rename "Http" section of Application settings

### ISSUE THREAD

**Issue by jsha** [keepassxreboot/keepassxc#286]
Right now, to enable integration with browser extensions, you need to go into Tools > Settings > Http. In order for a user to know this, they have to know internal implementation details of the KeePassXC <--> browser integration, specifically that the integration occurs over HTTP.

## Possible Solution
I propose changing the "Http" section of Application Settings to "browser integration."

## Context
I'm trying to provide password manager advice to non-technical users.

## Your Environment
* KeePassXC version: 2.1.1

**Comment by phoerious** [comment]
I plan on redesigning the settings completely for 2.2.0 and will consider this proposal.

**Comment by technodrome** [comment]
I'm using 2.1.1 on Mac but cannot find this setting. Does really have to be this complicated?

**Comment by TheZ3ro** [comment]
@technodrome what version of Mac do you have and from where do you installed it? 
KeePassXC release for Mac < 10.12 doesn't have the HTTP plugin enabled

**Comment by technodrome** [comment]
©TheZ3ro directly from [here](https://github.com/keepassxreboot/keepassxc/releases). Latest release. I don't know what the policy is here, but user should make this decision IMHO if it is disabled during build.

**Comment by TheZ3ro** [comment]
@technodrome currently HTTP is disabled (at compile time) because it's not working on Mac < 10.12 like stated on the website: "Binary package with KeePassHTTP needs OS X 10.12" https://keepassxc.org/download#mac
We are working on fixing this #265

**Comment by phoerious** [comment]
You can download the build with or without HTTP, but the one with HTTP will not run on OS X old than Sierra. That's a problem with libmicrohttpd which we haven't been able to solve yet.

**Comment by technodrome** [comment]
Thanks for an update guys. My bad, I did not notice +HTTP version. I am on Sierra, it is working, just royally bothersome to use. It pops dialogs every time asking to allow access and if I check **always allow access** then it stops working altogether. The firefox extension shows warnings that keepass access was not properly configured but if I reconfirm, nothing happens - passwords will get prefilled only for those websites, where I manually allowed access before checking that "always allow access". Ahh, either I'm missing the overall philosophy here, or it's a mess.

Anyway thanks for really cool instantaneous answers. Highly, much appreciated. Love to see you're actively developing it and if these errors are ironed out I will be sure to send you some beer money. Ah, I will do so anyway, what the heck.
